verification/curator-service/ui/src/components/new-case-form-fields/Transmission.tsx

```diff
@@ -1,30 +1,54 @@
+import { Chip, makeStyles } from '@material-ui/core';
+
 import CaseFormValues from './CaseFormValues';
 import ChipInput from 'material-ui-chip-input';
 import FieldTitle from '../common-form-fields/FieldTitle';
 import { FormikAutocomplete } from '../common-form-fields/FormikFields';
 import React from 'react';
 import Scroll from 'react-scroll';
-import { makeStyles } from '@material-ui/core';
+import axios from 'axios';
 import { useFormikContext } from 'formik';
 
 const useStyles = makeStyles(() => ({
     fieldRow: {
         marginBottom: '2em',
         width: '100%',
     },
+    chip: {
+        margin: '0.5em',
+    },
+    section: {
+        marginBottom: '1em',
+    },
 }));
 
-interface SelectFieldProps {
-    name: string;
-    label: string;
-    values: (string | undefined)[];
-}
-
 export default function Transmission(): JSX.Element {
-    const { setFieldValue, setTouched, initialValues } = useFormikContext<
-        CaseFormValues
-    >();
+    const {
+        setFieldValue,
+        setTouched,
+        initialValues,
+        values,
+    } = useFormikContext<CaseFormValues>();
+    const [
+        commonPlacesOfTransmission,
+        setCommonPlacesOfTransmission,
+    ] = React.useState([]);
     const classes = useStyles();
+
+    React.useEffect(
+        () => {
+            axios
+                .get('/api/cases/placesOfTransmission?limit=5')
+                .then((response) =>
+                    setCommonPlacesOfTransmission(
```

Comment:
Is it safe to use this React-generated function, which operates on state, with `[]`?

https://reactjs.org/docs/hooks-faq.html#is-it-safe-to-omit-functions-from-the-list-of-dependencies

Reply:
Yes, because it sets the state, but doesn't depend on the value of state. We want this to only be called once on component creation.